src/templates/ssr/fabric-custom/index.ts

```diff
@@ -1,10 +1,18 @@
 import { post } from '$lib/messenger';
 
+const CACHE_BUST = '%%CACHEBUSTER%%';
 const DapAssetsRoot = `https://s3-eu-west-1.amazonaws.com/adops-assets/dap-fabrics`;
 const DapAssetsFolder: string = '[%DapAssetsFolder%]';
 
 const DapAssetsPath = `${DapAssetsRoot}/${DapAssetsFolder}`;
 const ThirdPartyTag: string = '[%ThirdPartyTag%]';
+const TrackingPixel: string | undefined = '[%TrackingPixel%]';
+const ResearchPixel: string | undefined = '[%ResearchPixel%]';
```

Comment:
Not a blocker, just for my learning: would these ever be `undefined`?

Reply:
Ah that was just me testing something! Will remove